modules/base/src/components/Support.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+
+  <div class="support">
+    <div class="ui basic segment">
+
+      <h2>Support</h2>
+
+      <p>
+        Community Support is available. As soon as we start with the SPN, customers will receive direct priority support.
+      </p>
+
+      <h3>Get Help</h3>
+
+      <p>
+        You can ask for help on our <a href="https://www.reddit.com/r/safing">Subreddit r/safing</a>.
+      </p>
+
+      <h3>Report Bugs</h3>
+
+      <p>
+        Please report bugs in the User Interface in the <a href="https://github.com/safing/portmaster-ui/issues">safing/portmaster-ui Github repo</a>, and bugs in the Portmaster Core in the <a href="https://github.com/safing/portmaster/issues">safing/portmaster Github repo</a>.
+      </p>
```

Comment:
Maybe mention the actual portmaster repository for firewalling/service bugs as well

Reply:
There is a second link ;)

Reply:
Lol, I missed the scroll bar :D